uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/initialise.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = function (casper, scenario) {
+  var config = require('../site-config');
+  var loginFormSelector = 'form#user-login-form';
+  var credentials = config.credentials[scenario.credential];
+
+  casper.echo('--------------------------------------------', 'COMMENT');
+  casper.echo('RUNNING SCENARIO: ' + scenario.label, 'PARAMETER');
+
+  if (scenario.performLogin) {
```

Comment:
https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/190972313/Javascript#Javascript-Guardclauses

Reply:
Thanks for catching this.